src/main/java/org/gridsuite/sensitivityanalysis/server/service/ActionsService.java

```diff
@@ -57,20 +57,24 @@ public Integer getContingencyCount(List<UUID> uuids, UUID networkUuid, String va
         return restTemplate.exchange(actionsServerBaseUri + path, HttpMethod.GET, null, Integer.class).getBody();
     }
 
-    public List<Contingency> getContingencyList(UUID uuid, UUID networkUuid, String variantId) {
-        Objects.requireNonNull(uuid);
+    public ContingencyListExportResult getContingencyList(List<UUID> uuids, UUID networkUuid, String variantId) {
```

Comment:
uuids is not a good parameter name...
Why not contingencyListUuids so that we know what it is?

Reply:
doen